react-design/my-app/src/ItemName.js

```diff
@@ -0,0 +1,7 @@
+export default function ItemName(props) {
+  const namesArray = props.items.map((animal) => {
+    return <span key={animal}>{animal}</span>;
+  });
+
+  return <div>{namesArray[props.index]}</div>;
```

Comment:
To avoid the extra work, you could tweak this so it only creates one span for the value at `props.index`.